src/components/CohortBuilder/ParticipantsTableView/SaveSetModal.tsx

```diff
@@ -107,14 +128,46 @@ const SaveSetModal: FunctionComponent<Props> = (props) => {
   const onFinish = async (values: Store) => {
     const { nameSet } = values;
 
-    await onCreateSet({
-      tag: nameSet,
-      userId: user.egoId,
-      api: api,
-      sqon: sqon,
-      onSuccess: onSuccessCb,
-      onNameConflict: onNameConflictCb,
-    });
+    switch (saveSetActionType) {
+      case SaveSetActionsTypes.EDIT:
+        if (setToRename) {
```

Comment:
I think you can merge this condition with the switch condition => case (SaveSetActionsTypes.EDIT && setToRename)

it would remove an indentation depth

Reply:
i dont think that would work.  i can change it to this. 
case SaveSetActionsTypes.EDIT:
        if (!setToRename) {
          break;
        }
        await onEditSet({